server/controllers/plugins.js

```diff
@@ -0,0 +1,99 @@
+const asyncHandler = require("express-async-handler");
+const Card = require("./../models/Card");
+const BasePlugin = require("../models/BasePlugin");
+
+exports.activate = asyncHandler(async (req, res) => {
+  const { pluginName } = req.params;
+  const response = await BasePlugin.attach(Card, pluginName);
+
+  if (response.status === 400) {
+    res.status(400).json({ message: response.message });
+  }
```

Comment:
You know what we could actually do. `res.status(response.status).json(response.data)`

Thoughts?

Reply:
An easy way to handle multiple responses from the plugin.

Reply:
Ok we can work with that, makes it more generic.